### DISCUSSION TITLE
List of CI checks

### DISCUSSION THREAD

**Discussion by savchenko** [taskchampion/taskchampion#219] [General]
Re. https://github.com/taskchampion/taskchampion/issues/140, what should tested as the part of CI run? So far:

1. Documentation build from `.md` files
2. GH-actions "matrix" https://github.com/taskchampion/taskchampion/issues/216#issuecomment-828447108 to test against minimum supported and stable Rust versions.
3. Task/Task-server compilations

Please feel free to expand further.

**Comment by djmitche** [comment]
Aside from the matrix, I think this is all tested already.  We also run the full test suite (implying compilation) and a `cargo audit`.
